client/my-sites/stats/stats-banners/index.jsx

```diff
@@ -53,7 +55,16 @@ class StatsBanners extends Component {
 	}
 
 	renderBanner() {
-		if ( this.showUpworkBanner() ) {
+		if ( this.props.isRecurringPaymentsStatsNudgeVisible ) {
+			return (
+				<RecurringPaymentsStatsNudge
+					siteSlug={ this.props.slug }
+					siteId={ this.props.siteId }
+					visible={ true }
```

Comment:
I don't think `visible` is used as a prop anywhere in `RecurringPaymentsStatsNudge`.

Reply:
Thanks, done!